Cargo.toml

```diff
@@ -31,6 +31,10 @@ tokio = { version = "1", features = ["fs", "time", "rt"] }
 futures = "0.3" # so the doctest for wrap_stream is nice
 pretty_assertions = "1.4.0"
 
+[target.'cfg(unix)'.dependencies]
+libc = "0.2"
+lazy_static = "1.4.0"
```

Comment:
Please forgive me for chiming in, but `lazy_static` is effectively [deprecated](https://github.com/rust-lang-nursery/lazy-static.rs/issues/214). Maybe use [`once_cell::sync::Lazy`](https://docs.rs/once_cell/latest/once_cell/sync/struct.Lazy.html) or [`OnceLock`](https://github.com/rust-lang-nursery/lazy-static.rs/pull/216) instead?

Reply:
@smoelius exactly what I was going to suggest.